install_config/install/advanced_install.adoc

```diff
@@ -784,6 +783,44 @@ openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com
 ----
 
+[[advanced-install-configuring-registry-location]]
+=== Configuring the Registry Console
+
+If you are using a registry console image other than the default or require a specific version of the console,
+specify the desired registry within the
+*_/etc/ansible/hosts_* file. 
+
+----
+openshift_cockpit_deployer_prefix=<registry-name>/<namespace>/
+openshift_cockpit_deployer_basename=<my-console>
```

Comment:
This PR is targeted for 3.7, right? If yes, then it is okay. "openshift_cockpit_deployer_basename" is not supported yet in 3.6 openshift-ansible code yet.